vars/main.yml

```diff
@@ -14,15 +14,21 @@ k8s:
 core:
   standalone: "true"
   data_iface: ens18
+  # If upf.mode set to 'dpdk' then use below file for 'values_file',
+  # "deps/5gc/roles/core/templates/sdcore-5g-sriov-values.yaml"
   values_file: "deps/5gc/roles/core/templates/sdcore-5g-values.yaml"
-  ran_subnet: "172.20.0.0/16"
+  ran_subnet: "172.20.0.0/16"	# set to empty string to get subnet from 'data_iface'
   helm:
     local_charts: false
     chart_ref: aether/sd-core
     chart_version: 1.0.4
   upf:
     ip_prefix: "192.168.252.0/24"
     iface: "access"
+    # when using 'dpdk' mode please make sure at least two VF devices are
+    # created out of 'data_iface' and these devices are attached to
+    # vfio-pci driver
+    mode: af_packet          # Options: af_packet or dpdk
```

Comment:
```suggestion
    # when using 'dpdk' mode please make sure at least two VF devices are
    # created out of 'data_iface' and these devices are attached to
    # vfio-pci driver
    # If mode set to 'dpdk', use 'sdcore-5g-sriov-values.yaml' file.
    # i.e., 'values_file: "deps/5gc/roles/core/templates/sdcore-5g-sriov-values.yaml"'
    mode: af_packet          # Options: af_packet or dpdk
```

Reply:
Alternatively, what about only setting the `mode` variable in the `sriov` blueprint and remove it from here such that its default value is `af_packet` (if not set)? So, we can remove the whole comment from here.
@llpeterson any thoughts/preference on this?

Reply:
I'm ok making requiring  that mode be set for both cases, with mode = af_packet by default. (I plan to make some cosmetic changes to the comments to this doesn't confuse a new user.

Reply:
I feel it would be better to keep the mode parameter in both files (with af_packet default). It will help the user to know there is an option to change mode for UPF (instead of going to check reference files/wiki every time)